### PR TITLE
DX-321 Updated ME035 + ME039 formats from Money to Integer

### PR DESCRIPTION
**Pull request by basiq-ash**

### CODE REVIEW

**Review by jayden-basiq — CHANGES_REQUESTED** [review]

File: reports.yml
[the remark this answers is not in the document]
Comment (basiq-ash) [reply reports.yml:1]: [Switched to Positive value and format as Interger](https://github.com/basiqio/api-ref/pull/199/commits/5391bafd63538106d06372f2d964c6c92f90cd7f)

**Review by jayden-basiq — CHANGES_REQUESTED** [review]
Sorry to be a stickler @ashmanmalik 

ME039 is a number of months that Income has been stable for it would make more sense to have a smaller integer just to avoid confusion 

ie 
- id: ME035
                    title: Total Income has been stable for (months)
                    description: Number of months within report over which total income has been stable
                    sections:
                    - Expenses
                    result:
                      value: "3"
                      format: integer

**Comment by basiq-ash** [comment]
@jayden-basiq [DX-321 Updated ME035 Month Value](https://github.com/basiqio/api-ref/pull/199/commits/57516d9e6f99611d75cb2ec7c8dcb9b2c2a6ffc6)

File: reports.yml
[the remark this answers is not in the document]
Comment (basiq-ash) [reply reports.yml:1]: Resolved.

**Review by jayden-basiq — APPROVED** [review]
LGTM

**Pull request merged**